contracts/adapters/FlexibleLeverageStrategyExtension.sol

```diff
@@ -36,7 +36,7 @@ import { StringArrayUtils } from "../lib/StringArrayUtils.sol";
 
 
 /**
- * @title FlexibleLeverageStrategyAdapter
+ * @title FlexibleLeverageStrategyExtension
  * @author Set Protocol
  *
  * Smart contract that enables trustless leverage tokens using the flexible leverage methodology. This adapter is paired with the CompoundLeverageModule from Set
```

Comment:
Nit: there's a couple instances of using `adapter` in the javadoc. Let's just update those to `extension`